Firmware/LowLevel/platformio.ini

```diff
@@ -18,12 +18,6 @@ platform = https://github.com/maxgerhardt/platform-raspberrypi.git
 board = pico
 framework = arduino
 
-; Latest (known) version which compiles well with jpiat/PioSPI.
-; framework-arduinopico >= 4.4.3 will have a (PIO-Assembly driven) SoftwareSPI implementation which would make jpiat/PioSPI needless,
-; but I currently have no mainboard with LSM6DSO here for testing. So stick to the last working version till someone can switch & test it.
-platform_packages =
-    framework-arduinopico @ https://github.com/earlephilhower/arduino-pico.git#4.3.0
```

Comment:
Maybe we should pin this to the current version? @Apehaenger WDTY?

Reply:
Or next year may have the same issue where not much has landed here over the winter so CI didn't run, then we find that builds are broken in the spring